front/src/applications/operationalStudies/components/MacroEditor/export.ts

```diff
@@ -0,0 +1,114 @@
+import { osrdEditoastApi, type TrainScheduleBase } from 'common/api/osrdEditoastApi';
+import type { AppDispatch } from 'store';
+
+import type { NetzgrafikDto, NGETrainrunEvent, TrainrunSection, Node, Trainrun } from './types';
+
+const CREATED_TRAINRUN = new Map<number, number>();
```

Comment:
Tiny nit: since this is a map that we mutate at runtime (adding/removing elements as NGE operations are handled), it's not really a constant value, so maybe it's unfair to put it in UPPER_CASE?

Reply:
Done